backend/apigateway/src/main/java/pt/ulisboa/tecnico/socialsoftware/apigateway/config/ScheduledTasks.java

```diff
@@ -1,40 +0,0 @@
-package pt.ulisboa.tecnico.socialsoftware.apigateway.config;
-
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.scheduling.annotation.Scheduled;
-import org.springframework.stereotype.Component;
-import pt.ulisboa.tecnico.socialsoftware.tournament.demoutils.TournamentDemoUtils;
-import pt.ulisboa.tecnico.socialsoftware.tutor.answer.AnswerService;
-import pt.ulisboa.tecnico.socialsoftware.tutor.demoutils.TutorDemoUtils;
-import pt.ulisboa.tecnico.socialsoftware.tutor.impexp.ImpExpService;
-
-@Component
-public class ScheduledTasks {
```

Comment:
Can these tasks continue to occur?

Reply:
Each service has its own scheduled tasks now